benches/payout-stakers/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-01-19
```

Comment:
According to this: https://github.com/Cardinal-Cryptography/aleph-node/pull/319/files we want only one such file in the repository.

Reply:
Good catch, looks like this feature checks recursively filesystem up to root repo level, which makes sense. I'll remove this file.